recipes/kikuchipy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "kikuchipy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf55afb5441ce300399e275a31ac88d8381a0dd02afa000c788b3dc86151ae8e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```

Comment:
I think this recipe can be `noarch`, so please remove the `skip` line and add `noarch: python` instead.

Reply:
Thanks @npavlovikj, made the changes you suggested.